opengrok-indexer/src/main/java/org/opengrok/indexer/web/Scripts.java

```diff
@@ -38,6 +38,12 @@
  */
 public class Scripts implements Iterable<Scripts.Script> {
 
+    private static final String DEBUG_SUFFIX = "-debug";
+
+    enum ScriptType {
+        MINIFIED, DEBUG
+    }
```

Comment:
this could be a separate class or the naming could be just Type as when it's nested it's sort of redundant